--- ./env/Lib/site-packages/arcade/pymunk_physics_engine.py	2022-10-13 17:01:07.416779700 +1000
+++ ./pymunk_physics_engine.py	2022-10-13 16:47:13.638362100 +1000
@@ -311,7 +311,7 @@
 
         def _f3(arbiter, space, data):
             sprite_a, sprite_b = self.get_sprites_from_arbiter(arbiter)
-            pre_handler(sprite_a, sprite_b, arbiter, space, data)
+            return pre_handler(sprite_a, sprite_b, arbiter, space, data)
 
         def _f4(arbiter, space, data):
             sprite_a, sprite_b = self.get_sprites_from_arbiter(arbiter)
